modules/rostests/apitests/ntdll/NtFreeVirtualMemory.c

```diff
@@ -10,7 +10,7 @@ static void Test_NtFreeVirtualMemory(void)
     PVOID Buffer = NULL, Buffer2;
     SIZE_T Length = PAGE_SIZE;
     NTSTATUS Status;
-    
+
```

Comment:
Why?

Reply:
Indentation of blank lines seems "unusual".
I split that part out in a second commit: to be squashed or discarded?